bucket_map/src/bucket_storage.rs

```diff
@@ -226,12 +226,14 @@ impl<O: BucketOccupied> BucketStorage<O> {
         unsafe { slice.get_unchecked_mut(0) }
     }
 
+    #[allow(dead_code)]
```

Comment:
these will be used again shortly when we move refcount to a header on the data buckets.